config/locales/en.yml

```diff
@@ -1,19 +1,43 @@
+---
 en:
-  hello: 'Hello world'
-  title: 'Title'
-  log: 'LOG'
-  goals: 'GOALS'
-  tasks: 'TASKS'
-  profile: 'PROFILE'
-  new_entry: 'Add New Log Entry'
-  create_entry: 'Create New Entry'
-  welcome: 'Welcome to journey!'
-  journey: 'Journey.'
+  alerts:
+    login:
+<<<<<<< HEAD
+      failed: Login Failed
+      succesfull: Login Succesfull
+  create_entry: Create New Entry
+  create_user: Register New User
   errors:
-    no_title: 'Title is required.'
-    creating_failed: 'There was an error creating the journal entry.'
+    creating_failed: There was an error creating the journal entry.
+    no_title: Title is required.
+  goal:
+    create: Create New Goal
+    new: Add New Goal
+  goals: GOALS
+  hello: Hello world
+  journey: Journey.
+  log: LOG
+  login_user: Login
+  new_entry: Add New Log Entry
   placeholders:
-    entries: 'Entries View'
-    goals: 'Goals View'
-    tasks: 'Tasks View'
-    profile: 'Profile View'
+    entries: Entries View
+    goals: Goals View
+    profile: Profile View
+    tasks: Tasks View
+  profile: PROFILE
+  tasks: TASKS
+  title: Title
+  welcome: Welcome to journey!
+=======
+      succesfull: 'Login Succesfull'
+      failed: 'Login Failed'
+<<<<<<< HEAD
+<<<<<<< HEAD
+      
+>>>>>>> parent of e857b13 (create new usw)
+=======
+      
+>>>>>>> parent of e857b13 (create new usw)
+=======
+      
+>>>>>>> parent of e857b13 (create new usw)
```

Comment:
?

Reply:
i first commited and pushed the changes in the wrong branch, undid them and put them into the correct branch this must have happend there. i deleted it.